packages/qwik/src/core/v2/client/vnode-diff.ts

```diff
@@ -592,6 +598,9 @@ export const vnode_diff = (
             HANDLER_PREFIX + ':' + scope + ':' + eventName,
             value
           );
+          if (eventName) {
+            registerQwikLoaderEvent(eventName);
```

Comment:
So we don't need to add the scope to the event name, right?

Reply:
Yes